unwind/src/lib.rs

```diff
@@ -96,28 +96,26 @@ struct UnwindInfo<R: Reader> {
     personality: Option<Pointer>,
     lsda: Option<Pointer>,
     initial_address: u64,
-    ctx: UninitializedUnwindContext<EhFrame<StaticReader>, StaticReader>,
 }
 
 impl ObjectRecord {
-    fn unwind_info_for_address(&self,
-                               ctx: UninitializedUnwindContext<EhFrame<StaticReader>, StaticReader>,
-                               address: u64) -> gimli::Result<UnwindInfo<StaticReader>> {
+    fn unwind_info_for_address(
+        &self,
+        ctx: &mut UninitializedUnwindContext<StaticReader>,
+        address: u64,
+    ) -> gimli::Result<UnwindInfo<StaticReader>> {
         let &ObjectRecord {
             ref eh_frame_hdr,
-            eh_frame: ref sel,
+            ref eh_frame,
             ref bases,
             ..
         } = self;
 
         let fde = eh_frame_hdr.table().unwrap()
-            .lookup_and_parse(address, bases, sel.clone(), |offset| sel.cie_from_offset(bases, offset))?;
-
+            .fde_for_address(eh_frame, bases, address, |eh_frame, bases, offset| eh_frame.cie_from_offset(bases, offset))?;
```

Comment:
```suggestion
            .fde_for_address(eh_frame, bases, address, EhFrame::cie_from_offset)?;
```

Reply:
Didn't notice the signatures were the same :)